src/garage/envs/bullet/bullet_env.py

```diff
@@ -1,65 +1,69 @@
 """Wrappers for py_bullet environments."""
 import inspect
 
-import akro
 import gym
-from pybullet_envs.bullet.minitaur_duck_gym_env import MinitaurBulletDuckEnv
-from pybullet_envs.bullet.minitaur_gym_env import MinitaurBulletEnv
-from pybullet_envs.env_bases import MJCFBaseBulletEnv
 
-from garage.envs.env_spec import EnvSpec
+from garage.envs import GymEnv
 
+# The bullet gym environments in this list inherit from the
+# MJCFBasedBulletEnv class, which doesn't follow a consistent class variable
+# naming practice -> constructor param `render` is stored as `isRender`. Thus
+# they require additional pickling logic.
+_MJCF_BASED_BULLET_ENVS = [
+    'ReacherBulletEnv', 'PusherBulletEnv', 'StrikerBulletEnv',
+    'ThrowerBulletEnv', 'Walker2DBulletEnv', 'InvertedPendulumBulletEnv',
+    'InvertedDoublePendulumBulletEnv', 'InvertedPendulumSwingupBulletEnv',
+    'HalfCheetahBulletEnv', 'AntBulletEnv', 'HopperBulletEnv',
+    'HumanoidBulletEnv', 'HumanoidFlagrunBulletEnv',
+    'HumanoidFlagrunHarderBulletEnv'
+]
 
-class BulletEnv(gym.Wrapper):
+
+class BulletEnv(GymEnv):
     """Binding for py_bullet environments."""
 
-    def __init__(self, env=None, env_name='', is_image=False):
-        """Returns a Garage wrapper class for bullet-based gym.Env.
+    def __new__(cls, *args, **kwargs):
```

Comment:
this just calls `super()` -- can't it be removed?

Reply:
If this is removed a `RecursionError` will occur as it is using GymEnv's `__new__`